src/main/java/org/cryptomator/ui/keyloading/hub/DeviceAlreadyExistsException.java

```diff
@@ -0,0 +1,6 @@
+package org.cryptomator.ui.keyloading.hub;
+
+/**
+ * Thrown, when Hub registerDevice-Request returns with 409
+ */
+public class DeviceAlreadyExistsException extends RuntimeException {}
```

Comment:
We have a checked exception type for errors during unlock. I believe its name is sth like MasterkeyLoadingFailed